packages/input-option/src/InputOption/InputOption.types.ts

```diff
@@ -42,6 +77,17 @@ export interface BaseInputOptionProps {
    * @default true
    */
   isInteractive?: boolean;
+
+  /**
+   * Styles input based on intended action
+   * @default 'default'
+   */
+  actionType?: ActionType;
+
+  /**
+   * Determines how the items are styled
```

Comment:
Can you add a default here?

Reply:
Can you add the description on [L15](https://github.com/mongodb/leafygreen-ui/pull/2150/files#diff-74aa5a26066cb7cef22f2f104153f3dc93379a26acdb577f026a3199ca61c0abR15) to this? An outsider looking in might not understand this.